demo/campaign/src/documents/pages/blocks/SeoBlock.tsx

```diff
@@ -14,30 +14,24 @@ export const SeoBlock: React.FunctionComponent<SeoBlockProps> = ({
 }) => {
     const usedHtmlTitle = htmlTitle && htmlTitle != "" ? htmlTitle : title;
     return (
-        <>
-            <Head>
-                <title>{usedHtmlTitle}</title>
+        <Head>
+            <title>{usedHtmlTitle}</title>
```

Comment:
For future PR's:
This block shouldn't be needed here. SEO does not make sense in emails. I'll create a task